tests/test_cache.py

```diff
@@ -123,38 +122,6 @@ def test__copy_partial_parse_to_project_msg_fails_msgpack(mock_unpack, tmp_path,
     assert "Unable to patch the partial_parse.msgpack file due to ValueError()" in caplog.text
 
 
-def test__create_folder_version_hash(tmp_path, caplog):
```

Comment:
moved this test as is to `tests/test_versioning.py` module corresponding to the source method movement